JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -76,11 +79,14 @@ module Microsoft.ApplicationInsights.Context {
             // renew if acquisitionSpan or renewalSpan has ellapsed
             if (acquisitionExpired || renewalExpired) {
                 // update automaticSession so session state has correct id                
-                this.automaticSession.isFirst = undefined; 
+                this.automaticSession.isFirst = undefined;
                 this.renew();
```

Comment:
`renew()` is also calling `setCookie()` ultimately, and doesn't seem to be guarded by the same "only update cookie every X milliseconds logic."

Rather than manual `new Date()` checking, would it be possible to wrap the entire `update` function in a throttled function? For instance you could use Lodash `_.throttle` or just `setTimeout`/`setInterval` to achieve this. E.g.:

```js
this.update = _.throttle(originalUpdateFunction.bind(this), 60000);
```